index.js

```diff
@@ -0,0 +1,115 @@
+const URL = 'https://trektravel.herokuapp.com/trips';
+
+
+const reportStatus = (message, alertClass) => {
+    $('#status-message').removeClass();
+    $('#status-message').addClass(alertClass);
+    $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+    let content = `<p>${message}</p>`
+    content += "<ul>";
+    for (const field in errors) {
+      for (const problem of errors[field]) {
+        content += `<li>${field}: ${problem}</li>`;
+      }
+    }
+    content += "</ul>";
+    reportStatus(content, "alert alert-warning");
+  };
+
+const clearForm = () => {
+    $('#trip-form').get(0).reset();
+};
+
+
+
+const loadTrips = () => {
+    reportStatus('Loading trips...');
+
```

Comment:
Looks like your editor is set up to make 4-space tabs in JavaScript. At Ada we recommend 2-spaces, which will be important when you start working on pair projects. Please update this setting.